mw-webapp/cypress/fixtures/homeFixture.json

```diff
@@ -0,0 +1,4 @@
+{
+    "title": "Accelerate your goals with an asynchronous learning app",
+    "btnTitle": "Start for free!"
```

Comment:
I don't know bur what will be if developers will change button value on frontend from "Start for free!" to "Something other". Is it will be okey or will be needed to update these texts too?  Is it possible to use our dictionaries .json?

Reply:
Изучу этот вопрос и исправлю.

Reply:
done